provisioning/miniprov/hare_mp/main.py

```diff
@@ -201,6 +202,97 @@ def test(args):
         exit(-1)
 
 
+def reset(args):
+    try:
+        rc = 0
+        exit(rc)
```

Comment:
It's not great that every small function can suddenly perform exit(). Small function does some small job but exti() looks like something existential. Usually it is good to have just one single point in the code that makes a decision whether the application should be finished and what exit code should it issue.

It was fine when the number of such functions was not great, but I don't like that this habit is growing.

But perhaps it should be addressed with a separate PR.
@mssawant What do you think?